src/FFMpeg/Format/Audio/DefaultAudio.php

```diff
@@ -41,6 +47,46 @@ public function getExtraParams()
     /**
      * {@inheritdoc}
      */
+    public function getEnableVbrEncoding()
+    {
+        return $this->vbrEncodingQuality;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function setEnableVbrEncoding($value)
+    {
+        $this->enableVbrEncoding = (bool) $value;
```

Comment:
I would add a test like this:
```php
if(is_bool($value)) {
    $this->enableVbrEncoding = (bool) $value;
}
else {
    $this->enableVbrEncoding = FALSE;
}
```

Just to make sure we're not trying to convert something which is not appropriate.